### PR TITLE
Bulgarian language updates

### PR DESCRIPTION
**Pull request by AndreiPervikov**
Finally I did it and submitted my work

### CODE REVIEW

File: svelte/src/libs/translations/languages/bulgarian [hunk 1]
```diff
@@ -0,0 +1,101 @@
+{
```
Comment (getneil) [inline svelte/src/libs/translations/languages/bulgarian:1]: this should be bg.json

Comment (AndreiPervikov) [reply svelte/src/libs/translations/languages/bulgarian]: check it now, job is done

**Comment by getneil** [comment]
@AndreiPervikov please fix the conflicts too.

**Comment by AndreiPervikov** [comment]
fixed

**Comment by getneil** [comment]
@AndreiPervikov please check from time to time conflicts are pretty common with files here.

**Comment by AndreiPervikov** [comment]
updated check it now

**Comment by AndreiPervikov** [comment]
how it is now? ok?

**Comment by AndreiPervikov** [comment]
is it ready for merge?

**Comment by AndreiPervikov** [comment]
Can you merge it?

**Comment by getneil** [comment]
> Can you merge it?

there is still conflict. maybe try to resolve it locally.

**Comment by AndreiPervikov** [comment]
hey can you check it?

**Comment by AndreiPervikov** [comment]
can you merge it?

**Comment by AndreiPervikov** [comment]
hey any updates? I see that all checks have passed

**Comment by AndreiPervikov** [comment]
conflicts is fixed

**Comment by AndreiPervikov** [comment]
why u ignore me?

**Comment by getneil** [comment]
@AndreiPervikov please fix the conflicts again. just check here from time to time but if you can resolve this fast you are next in the queue. its just that so many of you are editing the same file.

**Comment by AndreiPervikov** [comment]
fixed

**Comment by AndreiPervikov** [comment]
I will be glad if you check mine fork first

**Review by getneil — APPROVED** [review]

**Pull request merged**